bentoml/yatai/web/src/server.ts

```diff
@@ -193,50 +197,28 @@ const createRoutes = (app, yataiClient) => {
       return res.status(500).json(error);
     }
   });
-
-  return router;
 };
 
-export const getExpressApp = (grpcAddress: string | null, baseURL: string) => {
+export const getExpressApp = (grpcAddress: string | null) => {
   const app = express();
-  const cookieParser = require('cookie-parser');
-  app.use(cookieParser());
-  app.use(function (req, res, next) {
-    var cookie = req.cookies.cookieName;
-    if (cookie === undefined) {
-      res.cookie('baseURLCookie',baseURL, { maxAge: 900000, httpOnly: false });
-      console.log('cookie created successfully');
-    } else {
-      console.log('cookie exists', cookie);
-    }
-    next();
-  });
 
   app.use(express.json());
   app.use(
     morgan("combined", {
       stream: { write: (message) => logger.info(message.trim()) },
     })
   );
+  app.use(express.static(path.join(__dirname, "../dist/client")));
   const yataiClient = createYataiClient(grpcAddress);
-  const router = createRoutes(app, yataiClient);
-  const apiUrlPrefix = (baseURL=="." ? '/' : '/' + baseURL);
```

Comment:
Any reason we are removing the prefix app URL?

Reply:
@yubozhao do we have a test for that?

Reply:
We don't have a test for that atm.

Reply:
Is this something Mehdi can look into? Is it blocking submission of this PR?

Reply:
@flosincapite I think it would be great that @ CallmeMehdi can look into adding tests for Yatai web server. I think we can do it either in this PR or another one.

For my initial question, I think removing the prefix code will break the ability for the user-defined prefix path. It'd be great to confirm that feature won't be broken after this PR.